dotnet/src/support/Events/EventFiringWebDriver.cs

```diff
@@ -1,19 +1,20 @@
-// <copyright file="EventFiringWebDriver.cs" company="WebDriver Committers">
+// <copyright file="EventFiringWebDriver.cs" company="Selenium Committers">
 // Licensed to the Software Freedom Conservancy (SFC) under one
-// or more contributor license agreements. See the NOTICE file
+// or more contributor license agreements.  See the NOTICE file
```

Comment:
Double whitespace.

Reply:
Back in the olden days before the internet, 2 spaces was the standard. Regardless, every other file in Selenium repo has this information double spaced.

Reply:
Why not triple spaced.. :) Since it is spread across all files in the repo, I am ok with it.